app/components/preprint-title-editor.js

```diff
@@ -0,0 +1,45 @@
+import Ember from 'ember';
+
+import { validator, buildValidations } from 'ember-cp-validations';
+
+const TitleValidation = buildValidations({
+    nodeTitle: {
+        description: 'Title',
+        validators: [
+            validator('presence', true),
+            validator('length', {
+                // minimum length for title?
+                max: 200,
```

Comment:
Confirmed that this matches title validation on OSF https://github.com/CenterForOpenScience/osf.io/blob/943412fe45b208dde7068b401a3934d98b71a53b/website/project/model.py#L740